src/compo/modis_aod2ioda.py

```diff
@@ -41,46 +42,67 @@
 
 # Get the group names we use the most.
 metaDataName = iconv.MetaDataName()
-obsValName = iconv.OvalName()
-obsErrName = iconv.OerrName()
-qcName = iconv.OqcName()
 
+varsKeyList = [('valKey', iconv.OvalName(), 'float', 'longitude latitude', '1'),
+               ('errKey', iconv.OerrName(), 'float', 'longitude latitude', '1'),
+               ('qcKey', iconv.OqcName(), 'integer', 'longitude latitude', None)]
+
+float_missing_value = nc.default_fillvals['f4']
+int_missing_value = nc.default_fillvals['i4']
+double_missing_value = nc.default_fillvals['f8']
```

Comment:
not sure if these are any different, but typically use these from `iconv` in an attempt to keep them consistent between converters

```suggestion
float_missing_value = iconv.get_default_fill_val(np.float32)
int_missing_value = iconv.get_default_fill_val(np.int32)
long_missing_value = iconv.get_default_fill_val(np.int64)
```

Reply:
@srherbener do you know if `nc.default_fillvals['f4']` and `iconv.get_default_fill_val(np.float32)` are the same ?